### PR TITLE
Auto PR due to push to staging

### PR DESCRIPTION
**Pull request by ExC0d3**
Testing move of pr req script into utils.

### CODE REVIEW

**Review by gcallah — APPROVED** [review]
Moving the utility to utils seems A-OK!
Now, on to making it more generic.

File: lib/model.py [hunk 1]
```diff
@@ -305,16 +305,21 @@ def runN(self, periods=DEF_TIME):
...
 
     def add_child(self, group):
         """
         Put a child agent in the womb.
         group: which group will add new agent
+        The womb should move up into model eventually.
         """
         self.env.add_child(group)
 
+    def get_pop_hist(self):
```
Comment (ExC0d3) [inline lib/model.py:320]: If this is new there should be a test for this also?
Or is it already present?

**Review by ExC0d3 — COMMENTED** [review]
Missing test for new method in model.py?

File: lib/model.py (hunk 1 above)
Comment (gcallah) [reply lib/model.py:320]: Yes, I will write the test... but I wanted to get Troy's endpoint working.

**Review by gcallah — APPROVED** [review]
Looks OK, but where are new reviewers?

**Comment by ExC0d3** [comment]
> Looks OK, but where are new reviewers?

Probably need the env variable in travis as well?

**Comment by gcallah** [comment]
> > Looks OK, but where are new reviewers?
> 
> Probably need the env variable for reviewers file in travis as well?

Yes, eventually... but I put in a default file if the env var is not there that *ought* to work.

**Pull request merged**